src/status/status.py

```diff
@@ -33,6 +33,12 @@
 
 SERVICES_DICT = {}
 
+### SSL parameters
+USE_SSL = os.environ.get("F7T_USE_SSL", False)
```

Comment:
Ditto. I started to see a pattern, potential candidate for a refactoring :)

Reply:
Yes, there is not `bool()` casting in environment variables.